docs/video/obs/index.md

```diff
@@ -80,19 +79,23 @@ It's very useful for testing encoder settings (with lots of movement in the test
 
 You can configure it in the **Replay Buffer** tab in **Output**, and you can set hotkeys for it in **Hotkeys** after it has been enabled.
 
-## :material-account-question: Where to go from here
+## :material-account-question: What Next?
 
 !!! warning "Ranking"
     NVIDIA's encoder (NVENC) is by far the best for recording, as it is fast and not resource intensive, unless you push it to its limits. Other encoders may struggle to record very high FPS, such as 240 FPS and over.
 
-From here, you'll want to go to the page which fits [your GPU](https://www.microsoft.com/en-us/windows/learning-center/how-to-check-gpu "How to check your GPU in Windows").
+From here, you'll want to go to the page which fits [your GPU](https://www.microsoft.com/en-us/windows/learning-center/how-to-check-gpu "How to check your GPU in Windows"):
 
 - :simple-nvidia: [**NVIDIA NVENC**](/video/obs/nvenc)
 - :custom-amd: [**AMD AMF**](/video/obs/amf)
 - :simple-intel: [**Intel QuickSync**](/video/obs/quicksync)
 
+<<<<<<< Updated upstream
```

Comment:
might want to fix this lol

Reply:
There are a lot of these I forgot to fix, I'm not a fan of whatever vscode extension does that

Reply:
it's just git